network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/iframe_block.html

```diff
@@ -1,14 +1,44 @@
 {% load wagtailcore_tags %}
+{% comment %}
+  The iframe_block.html is used on Article pages and Blog pages.
+  Blog pages need to break out of the current grid system and open a 12 column block
+  for it to align with other content.
+  The Article page doesn't require as much "breaking out" of the parent grid because there's
+  less nesting in the article pages. But we still need to break out of the grid, and
+  use a 12 column grid.
+{% endcomment %}
 
-<div class="my-default">
-  <div class="embed-responsive embed-responsive-16by9" {% if value.height %}style="height: {{value.height}}px"{% endif %}>
-    <iframe src="{{ value.url }}" class="embed-responsive-item"></iframe>
-  </div>
-  {% if value.caption %}
-  <p class="body-small mt-2">
-    {% if value.captionURL %}<a href="{{ value.captionURL }}">{% endif %}
-    {{ value.caption }}
-    {% if value.captionURL %}</a>{% endif %}
-  </p>
-  {% endif %}
-</div>
+{% if page_type == "blog" %}
+  {# Break out of the blog_page.html containment grid #}
+      </div> <!-- End .blog-body -->
+    </div> <!-- End .cms -->
+  </div> <!-- End .col-lg-8 -->
+{% elif page_type == "article" %}
+  {# Break out of the article_pag.html containment grid #}
+  </div> <!-- end .offset-sm-1.col-sm-10.col-12.article-blocks -->
+{% endif %}
```

Comment:
I think we should find a better way to do this, long term. How do we make it so we can have the same block across multiple templates without conditionals? Maybe some refactoring around where the bootstrap `.row` or `.container` classes are defined?

Reply:
This was my concern too when we decided to allow wider blocks (that break container) on certain page types. I can't think of other ways to build this without restructuring all our page template code. 😬

Reply:
@mmmavis Same here, I cant think of another way to do this other than to introduction a new stream block. Like a "blog_iframe" and a "publication_iframe". But becuse we're already using iframes, the migration would be tricky to write which we want to avoid if possible. So then we fall into the situation where there's an "iframe", "blog_iframe" and "publication_iframe" (where the normal "iframe" is on both paths, so there's 2 iframe blocks on each page, which gets confusing for editors). 

We could maybe rename the normal iframe to "normal iframe" and create a new one called "wide iframe" and split "wide iframes" into 2 files, one for publications and one for blog.

Both options are messy I think. Im happy to switch techniques in this PR if that's what we think is going to be most maintainable moving forward. Thoughts @mmmavis @benhohner ?